grails-app/services/org/pih/warehouse/report/ReportService.groovy

```diff
@@ -967,7 +967,7 @@ class ReportService implements ApplicationContextAware {
                 "PO Number"                                         : orderItem?.order?.orderNumber,
                 "Type"                                              : "Item",
                 "Code"                                              : orderItem?.product?.productCode,
-                "Description"                                       : orderItem?.product?.description,
+                "Description"                                       : orderItem?.product?.description ?: orderItem?.product?.displayNameOrDefaultName,
```

Comment:
I can see that in the ticket you were supposed to replace description with the product name whereas you do fallback to product name if the description is `null`.

Reply:
Oh, you're right. I was convinced that I had to do fallback to product name.

Reply:
Good catch @kchelstowski